tests/phpunit/includes/testcase-trait.php

```diff
@@ -242,9 +245,9 @@ protected function require_wp_menus( $trigger_hooks = true ) {
 	 * @return void
 	 */
 	protected static function create_model_legacy() {
-		$options                  = PLL_Install::get_default_options();
-		$options['hide_default']  = 0;
-		$options['media_support'] = 1;
-		self::$model              = new PLL_Admin_Model( $options );
+		$options = self::create_reset_options();
```

Comment:
Same comment as before :)

Reply:
Done in [37ee8c9](https://github.com/polylang/polylang/pull/1481/commits/37ee8c9e8a61d2efd62e24dd25c76c47e1f3333c).